.github/workflows/release.yml

```diff
@@ -103,3 +103,11 @@ jobs:
           docker push ${{ env.IMAGE_NAME }}:${{ needs.generate-version.outputs.version }}
           docker tag ${{ env.IMAGE_NAME }}:${{ needs.generate-version.outputs.version }} ${{ env.IMAGE_NAME }}:latest
           docker push ${{ env.IMAGE_NAME }}:latest
+
+      - name: Push to quay.io
+        run: |
+          echo "${{ secrets.QUAY_TOKEN }}" | docker login quay.io -u "${{ secrets.QUAY_USERNAME }}" --password-stdin
+          docker tag ${{ env.IMAGE_NAME }}:${{ needs.generate-version.outputs.version }} ${{ env.IMAGE_NAME_QUAY }}:${{ needs.generate-version.outputs.version }}
+          docker push ${{ env.IMAGE_NAME_QUAY }}:${{ needs.generate-version.outputs.version }}
+          docker tag ${{ env.IMAGE_NAME_QUAY }}:${{ needs.generate-version.outputs.version }} ${{ env.IMAGE_NAME_QUAY }}:latest
+          docker push ${{ env.IMAGE_NAME_QUAY }}:latest
```

Comment:
Do we want to keep offering the latest tag?

Reply:
I don't mind. Bad idea?

Reply:
Thought its recommended now to not use it, but to tell the truth: I use fixed version/hashs, if someone else wants to risk running into problems, then let them (unless to many bug reports pop up, or people think its our fault)